Topics/statistics-fundamentals.md

```diff
@@ -0,0 +1,167 @@
+# Statistics Fundamentals
+
+# What you will learn 
+
+In this guide, you'll learn what is the fundamental statistical knowledge you need to have in order
+to build insightful Data Science projects and build machine learning systems upon them.
+
+You'll also learn that not so much is needed to start to apply yourself to real-world data problems!
+
+## Prerequisites
+The previous Paradiso guides.
+
+## Time to complete
+10 minutes.
+
+# Index
+- [Motivation](#Motivation)
+- [Courses](#Courses)
+  - [Descriptive Statistics](#)
+  - [Probability Theory](#Probability-Theory)
+  - [Bayesian Statistics](#Bayesian-Statistics)
+- [Ask Questions](#Ask-Questions)
+- [Conclusions](#Conclusions)
+
+Let's dive right in!
+ 
+### Motivation
+The field of statistics is the **science of learning from data**.
+
+Statistics is a crucial process behind how we make discoveries in science, make decisions based on data, and make predictions. 
+
+Statistical knowledge helps you use the proper methods to collect the data, employ the correct analyses, and effectively present the results. 
+
+Every fancy term you read related to the field of Data Science, or the even more misunderstood Artificial Intelligence, _is nothing more than statistical models trained with some mathematical algorithms_.
+
+That's why **to learn the basics of statistics is fundamental** in order to tackle real-worlds Data Science problems and build powerful predictive models upon them.
+
+The two fundamental subjects to know in order to develop successful Data Science projects are **descriptive statistics** and **theory of probability**.
+
+We'll explore an additional one, Bayesian Statistics, which is pretty powerful but not mandatory for a first pass of the statistics topics.
+
```

Comment:
Can you add hyperlinks where possible in the whole guide - for Math and Stats related terms like Bayesian Statistics, etc...